library/EasyPipeline/src/Easy.hs

```diff
@@ -564,9 +564,11 @@ printTimeLapse byteString option = do
   --
   (_, cored) <- coreify byteString option
   traverse_ (printCoreFunction cored option) currentDefinedItems
+  print "finished Cored"
```

Comment:
remove this before merge? or do you want to keep the debugging info in the easy pipeline?

Reply:
This is useful to have for the timelapse as it came under question what a pass was.